Components/Services/PaymentService.php

```diff
@@ -642,7 +649,16 @@ private function configurePaymentSettings(PaymentInterface $paymentMethodObject,
         }
 
         if ($paymentMethodObject instanceof CreditCard) {
-            $ccToken = $this->creditCardService->getCardToken();
+            # prevent this method from failing, because at this
+            # point we're not sure if the credit card token
+            # is needed to finish the payment at Mollie
+            try {
+                $ccToken = $this->creditCardService->getCardToken();
+            } catch (CustomerNotFoundException $exception) {
+                $this->logger->error('Could not get the credit card token for the current customer.', [
```

Comment:
could you change that to a warning.
we dont really know what this is.
the token is optional only, and merchants are sometimes scared that things break on errors.
warning would make it softer :)